web/src/settings_emoji.ts

```diff
@@ -116,6 +116,7 @@ export function populate_emoji(): void {
         // Add people.js data for the user here.
         if (emoji.author_id !== null) {
             emoji.author = people.get_user_by_id_assert_valid(emoji.author_id);
+            emoji.author.is_active = people.is_person_active(emoji.author_id);
```

Comment:
What is this line doing? I would expect that `people.is_person_active` accesses the same singleton object that `people.get_user_by_id_assert_valid` will have returned, so this should be a noop.

Reply:
So it seems. I feel like that might not be the best way for `people.ts` to work; can you start a #frontend conversation about it?

Reply:
[CZO](https://chat.zulip.org/#narrow/stream/6-frontend/topic/Potential.20Enhancement.20for.20.60people.2Ets.60.20Functionality/near/1928931)

Reply:
It does no good to calculate whether the user is active once during `populate_emoji`, as activity changes over time. Instead, you need to calculate it within the `modifier_html` callback for the `$emoji_table` `ListWidget`.

Reply:
made the change